content/en/docs/tutorials/zerossl/zerossl.md

```diff
@@ -0,0 +1,203 @@
+
+
+Please note! \
+EAB credentials are not stored in your account, please make sure to note them somewhere. Each click on "Generate" will create a new set of credentials. Even if you create multiple credentials, all of them will remain functional.
+
+Please note! \
+EAB credentials are one-use only. Create additional pair for different environments.
+
+# Prerequisites
+
+- An AWS account
+- `kubectl` installed
+- Access to a publicly registered DNS zone
+- Kubernetes cluster, you can use AWS EKS
+- [`nginx-ingress`](https://kubernetes.github.io/ingress-nginx/) deployed and working inside cluster
+
+
+# Tutorial scenario:
+
+## Installing cert-manager
+
+There are no special requirements to note when installing cert-manager on EKS,
+so the regular [running on
+Kubernetes with helm](../../../installation/) guide
+can be used to install cert-manager.
+
+Please walk through the installation guide and return to this step once you
+have validated cert-manager is deployed correctly.
+
+In order to automatically switch to the `ZeroSSL` we recommend setting default shim by adding the following configuration to values file.
+
+```yaml
+ingressShim:
+  defaultIssuerName: "zerossl-production"
+  defaultIssuerKind: "ClusterIssuer"
+
+installCRDs: true
+```
+
+Install it using helm
+```
+helm upgrade  --install --namespace cert-manager  --version v1.5.3 cert-manager jetstack/cert-manager -f values.yaml 
+```
+
+
+## Configure your DNS records
+
+The best way to manage DNS using AWS is by using Route53. Create AWS account with permissions to modify Route53 rules.
+
+## EAB secret
+Once you will get your credentials first step is to create seed with secrets. They are responsible for authenticating with your `ZeroSSL` account. Encode it in base64 first.
+```bash
+echo -n "YOUR_ZEROSSL_EAB_SECRET" | base64 -w 0
+```
+
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: zero-sll-eabsecret
+data:
+  secret: YOUR_ENCODED_ZEROSSL_EAB_SECRET
+```
+```bash
+kubectl apply -f zero-sll-eabsecret.yaml -n cert-manager
+```
+
+### Second workaround of creating secret.
+
+```bash
+$ kubectl create secret generic \
+       zero-sll-eabsecret \
+       --namespace=cert-manager \
+       --from-literal=secret='YOUR_ZEROSSL_EAB_SECRET' \
+```
+
+## Cluster issuer
+Then we must create the `ZeroSSL` `ClusterIssuer`, let's call it `zerossl-production`. In our case we are using AWS. See pre-conditions to provision all required elements.
+
+```yaml
+apiVersion: cert-manager.io/v1alpha2
+kind: ClusterIssuer
+metadata:
+  name: zerossl-production
+spec:
+  acme:
+    # ZeroSSL ACME server
+    server: https://acme.zerossl.com/v2/DV90
+    email: dummy-email@yopmail.com
+
+    # name of a secret used to store the ACME account private key
+    privateKeySecretRef:
+      name: zerossl-prod
+
+    # for each cert-manager new EAB credencials are required
+    externalAccountBinding:
+      keyID: ZEROSSL_KEY_ID
+      keySecretRef:
+        name: zero-sll-eabsecret
```

Comment:
```suggestion
        name: zero-ssl-eabsecret
```

Reply:
Thanks @pascal-hofmann for pointing this out!